rust/noosphere/tests/gateway.rs

```diff
@@ -388,6 +390,60 @@ async fn gateway_receives_sphere_revisions_from_a_client() -> Result<()> {
     Ok(())
 }
 
+// #[tokio::test]
```

Comment:
missing a TODO, or could make as a pending test

Reply:
This test is included in this change, I just moved it somewhere else.